src/Symfony/Component/Console/Input/InputArgument.php

```diff
@@ -59,6 +59,9 @@ public function __construct(
         string|bool|int|float|array|null $default = null,
         private \Closure|array $suggestedValues = [],
     ) {
+        if ('command' === $name && 'The command to execute' !== $description) {
+            throw new InvalidArgumentException('Reserved keyword "command" was used as argument name.');
```

Comment:
I think this one could be rephrased to match the rest of the codebase, something like `The "command" keyword cannot be used as the argument name of a command.` maybe.

Reply:
Suggestion applied